doc/tools/docgram/fullGrammar

```diff
@@ -2566,6 +2566,20 @@ numnotoption: [
 | "(" "abstract" "after" bignat ")"
 ]
 
+numnotvial: [
+| reference reference
+| reference reference "," numnotvial
+]
+
+qualid__ind: [
```

Comment:
In the grammar, please don't use nonterminals with double underscores in them.  These names are ambiguous (in the doc, the tool can't tell distinguish whether it's the `qualid__ind` nonterminal or the `qualid` nonterminal with a subscript), which would break the automatic updating of commands and tactics in the rsts.

It's fine to introduce the subscript in the header here:

![image](https://user-images.githubusercontent.com/1253341/92024704-61408000-ed13-11ea-9f57-d87c8b2fd885.png)

The description will be easier to read if you add subscripts to the two `qualid`s in the box above and to each reference in the description below it.  As it is, the text is a bit ambiguous on what each `qualid` refers to.